src/interactive-window/interactiveWindow.ts

```diff
@@ -751,15 +755,10 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
             line: line,
             originalSource: code
         };
-        const id = uuid();
-        const generatedCode = this.codeGeneratorFactory
-            .getOrCreate(this.notebookDocument)
-            .generateCode({ interactive, id }, isDebug);
 
         const metadata: InteractiveCellMetadata = {
             interactiveWindowCellMarker,
             interactive,
-            generatedCode,
             id: uuid()
```

Comment:
Found a bug, the `id` passed into the generated code was wrong, as that's not whats stored in the metadata.
I.e. line 754 and 763 use two different ids.